keyboards/singa/info.json

```diff
@@ -0,0 +1,36 @@
+      },
+
+      "LAYOUT_WKL": {
```

Comment:
@mechmerlin or @noroadsleft would have to confirm, but I think the "standard" was to use all lower cases after LAYOUT.  So `LAYOUT_wkl`, etc. 

Also, wouldn't wkl be hhkb style? (or am I confused)

Reply:
HHKB is control key less so no bottom left key. 
WKL is winkeyless so no 2nd left key on bottom row. 

I don’t mind changing case tho, I just made stuff up, I don’t care what they are called.

Reply:
I always go full lowercase after `LAYOUT`, so:

- `LAYOUT`
- `LAYOUT_wkl`
- `LAYOUT_wkl_split_bs`
- `LAYOUT_tsangan`
- `LAYOUT_tsangan_split_bs`
- `LAYOUT_wk`
- `LAYOUT_wk_split_bs`

Note the underscore in `split_bs`, which is consistent with the naming scheme throughout the repo.

Also, I consider Winkeyless to be a  subset of Tsangan, so were it my board I wouldn't have either of the `_wkl` layouts. The only difference between those and the Tsangan layouts are the one missing key bottom left between Ctrl and Alt, but electrically there's no difference. Someone else might have a different opinion on that, though.

Reply:
The WKL is the ONLY layout supported by the WKL top plate. KC_NO in keymaps is gross and I don't want it. Yes, the Tsangan version would work, but then I'm stuck with KC_NO on every single map I use. This is better.

I'll split the _bs for you tho.

Reply:
> The WKL is the ONLY layout supported by the WKL top plate. KC_NO in keymaps is gross and I don't want it.

Fair.